BottomBar.Droid/Renderers/BottomBarPageRenderer.cs

```diff
@@ -227,7 +231,8 @@ void UpdateBarTextColor ()
 				return;
 			}
 
-			// haven't found yet how to set text color for tab items on_bottomBar, doesn't seem to have a direct way
+            _bottomBar.SetActiveTabColor(Element.BarTextColor.ToAndroid());

```

Comment:
Don't want to push this further. Perhaps the next time first welcome a new contributor. It's about motivation to contribute on a project.
I'm fine now, we found the reason.